projectile.el

```diff
@@ -1224,25 +1224,21 @@ The function calls itself recursively until all sub-directories
 have been indexed.  The PROGRESS-REPORTER is updated while the
 function is executing.  The list of IGNORED-FILES and
 IGNORED-DIRECTORIES may optionally be provided."
-  ;; we compute the ignored files and directories only once and then we reuse the
```

Comment:
Why did you remove this comment? It's kind of important. 😄

Reply:
My mistake, fixed